tests/drivers/clock_control/stm32_clock_configuration/stm32u5_core/boards/pll_hse_fracn_160.overlay

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2021 Linaro Limited
```

Comment:
Please use your own copyright.

Reply:
Thanks for the reminder, I will change the copyright later.